policy/modules/system/systemd.te

```diff
@@ -115,6 +115,28 @@ typealias systemd_generator_t alias { systemd_fstab_generator_t systemd_gpt_gene
 typealias systemd_generator_exec_t alias { systemd_fstab_generator_exec_t systemd_gpt_generator_exec_t };
 init_system_domain(systemd_generator_t, systemd_generator_exec_t)
 
+type systemd_homed_t;
+type systemd_homed_exec_t;
+init_daemon_domain(systemd_homed_t, systemd_homed_exec_t)
+
+type systemd_homework_t;
+type systemd_homework_exec_t;
+domain_type(systemd_homework_t)
+domain_entry_file(systemd_homework_t, systemd_homework_exec_t)
+role system_r types systemd_homework_t;
+
+type systemd_homed_runtime_t;
```

Comment:
I could not find and corresponding fc specs for this and possibly others. This may or may not affect any existing unix_stream_connect_userdbd() calls (as the socket may or may not have been mislabeled at the time)

Reply:
It should work fine without them, as I intentionally test with as few fcs as possible to make sure clean installs create files with correct labels. I forgot to add them after I finished.